lib/stronger_parameters/constraints/nil_string_contraint.rb

```diff
@@ -0,0 +1,15 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class NilStringConstraint < Constraint
+    NULL_VALUES = [nil, '', 'undefined', 'UNDEFINED']
```

Comment:
I think the only value that reasonable to happen is 'undefined' from
someone screwing up their js ... I'll check the logs for other usages ...

On Fri, Feb 13, 2015 at 1:04 AM, Pierre Schambacher <
notifications@github.com> wrote:

> In lib/stronger_parameters/constraints/nil_string_contraint.rb
> <https://github.com/zendesk/stronger_parameters/pull/5#discussion_r24651534>
> :
>
> > @@ -0,0 +1,15 @@
> > +require 'stronger_parameters/constraints'
> > +
> > +module StrongerParameters
> > +  class NilStringConstraint < Constraint
> > +    NULL_VALUES = [nil, '', 'undefined', 'UNDEFINED']
>
> What about Undefined? [image: :trollface:]
> Maybe we should check /undefined/i.match(v) ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/stronger_parameters/pull/5/files#r24651534>.
>